SECURITY.md

```diff
@@ -0,0 +1,63 @@
+# Security Policy
+
+This document describes the Security Policy for ApeWorX Ltd., including vulnerability disclosures for the [Ape Framework](https://github.com/ApeWorX/ape) and any of it's [maintained plugins](https://github.com/search?q=org%3Aapeworx+ape-+-repo%3AApeWorX%2Fape++-repo%3AApeWorX%2Fapeworx.github.io&type=Repositories).
```

Comment:
I just realized - should we have a policy that includes all repos we maintain, including `ethpm-types` and all that? If so, where would that live?

Reply:
Could add that here I suppose

Reply:
```suggestion
This document describes the Security Policy for ApeWorX Ltd., including vulnerability disclosures for the Ape Framework, any of it's maintained plugins, or any packages we maintain used by those as dependencies (see [here](](https://github.com/search?q=org%3Aapeworx+-repo%3AApeWorX%2Fapeworx.github.io&type=Repositories))).
```